disassemblers/ofrak_ghidra/ofrak_ghidra_test/conftest.py

```diff
@@ -24,7 +23,10 @@ def ghidra_components(ofrak_injector):
 async def hello_world_elf_resource(
     hello_world_elf, ofrak_context: OFRAKContext, test_id: str
 ) -> Resource:
-    resource = await ofrak_context.create_root_resource(test_id, hello_world_elf, tags=(Program,))
```

Comment:
Right, so that one was a bit awkward. Because the GhidraIdentifier checks for tags like `Elf`, it needs to run after the target is identified as Elf/PE/IHex otherwise the custom loader is used. Normally this just happens exactly like we want, because we start with a `GenericBinary` and only once it has already been identified as one of those tags would the GhidraIdentifier run on it. But here, because the file is explicitly tagged as a `Program` ahead of time, the GhidraIdentifier can run right away, same as the magic identifiers; they are at the same "depth" in the identification auto-run. Because of this, the tags applied by the other identifiers are not present when the GhidraIdentifier runs and the resource is incorrectly marked as a `GhidraCustomLoadProject`.

If the resource were tagged as `Elf` it would still work. It's the `Program` tag specifically that causes this problem.